gatorgrouper_django/gatorgrouper/models.py

```diff
@@ -0,0 +1,63 @@
+from django.db import models
+
+# Create your models here.
+class Professor(models.Model):
+    email = models.EmailField(max_length = 200)
+    professor_id = models.AutoField(primary_key=True)
+    first_name = models.CharField(max_length = 30)
+    last_name = models.CharField(max_length = 30)
+
+    def __str__(self):
+        return '{}'.format(self.email)
+
+
+class Semester_Class(models.Model):
+    professor_id = models.ForeignKey(
+            Professor,
+            on_delete=models.CASCADE,
+            )
+    semester = models.CharField(max_length = 50)
+    class_id = models.AutoField(primary_key=True)
+    department = models.CharField(max_length = 10)
+    class_number = models.CharField(max_length = 10)
+    class_section = models.CharField(max_length = 10)
+
+    def __str__(self):
+        return '{}: {}*{}'.format(self.department, self.class_number, self.class_section)
+
+
+class Assignments(models.Model):
+    class_id = models.ForeignKey(
+            Semester_Class,
+            on_delete=models.CASCADE,
+            )
+    assignment_id = models.CharField(max_length = 20, primary_key = True)
```

Comment:
@chenc-allegheny's assignment creation form template contains a text input for an assignment description, so I'd suggest adding an attribute for that (probably tweet length ish? idk)

Reply:
Just added a description attribute with a 250 char max length. 👍